src/app/components/sidebar/filters/Filters.tsx

```diff
@@ -2,28 +2,71 @@ import * as Styled from './Filters.styles';
 import { useIntl } from 'react-intl';
 import { ConditionsPanel } from './condition/ConditionsPanel';
 import { LabelWithButton } from 'ui';
-import { FiltersProps } from './Filters.types';
+import { FiltersProps, ActiveFliterItem } from './Filters.types';
+import { PricePanel } from './pricePanel/PricePanel';
+import { useLocale } from 'hooks';
+import { FormatNumberOptions } from 'react-intl';
+
+export const Filters = ({ onConditionClick, onPriceChange, activeFilters }: FiltersProps) => {
+  const { formatMessage, formatNumber } = useLocale();
+
+  const currencyConfig: FormatNumberOptions = {
+    style: 'currency',
+    currency: 'PLN',
+    maximumFractionDigits: 0,
+  };
+
+  const isAnyFilterApplied =
+    (activeFilters.condition && activeFilters.condition.length > 0) || activeFilters.minPrice || activeFilters.maxPrice;
+
+  const activeFliterItems: ActiveFliterItem[] = [];
```

Comment:
Tak jak już gdzieś wspominałem, staramy się nie mutować obiektów. Dodawanie elementów do tablicy jest mutacją. Spróbuj to samo zrobić tylko przy pomocy `map` i/lub `reduce`.
Druga rzecz - w tej linii masz dwie literówki ;)
Hint:
```
const activeFilterItems: ActiveFilterItem[] = [...activeFilters.condition ?? [], { name: 'minPrice', value: activeFilters.minPrice }, { name: 'maxPrice', value: activeFilters.maxPrice }].map(...
```
mniej więcej tak